### DISCUSSION TITLE
为什么小程序违规了？

### DISCUSSION THREAD

**Discussion by high-far** [YunYouJun/cook#17] [杂七杂八]
打不开了
发生了什么?

**Comment by YunYouJun** [comment]
- 小程序并非由我制作，与我无关，但应该是使用了本项目数据
- 似乎还存在另一个相同功能的小程序

---

因此，我猜测可能原因如下：

- 菜谱数据多导流 B 站，由于微信封闭生态，微信官方不希望有过多导流到 B 站的内容而封禁
- 被同行举报了（可以排除我）

以上纯属个人猜测，以供讨论。如有误，可提醒我删除。
